crates/ruff_linter/src/rules/flake8_annotations/rules/definition.rs

```diff
@@ -523,7 +523,7 @@ fn check_dynamically_typed<F>(
             if type_hint_resolves_to_any(
                 parsed_annotation.expression(),
                 checker,
-                checker.settings.target_version.minor(),
+                checker.settings.target_version.minor,
```

Comment:
Nit: A bit late now but it could have been worth adding a `minor` method to reduce the diff size.